### PR TITLE
Merge pre-release into Develop

### PR DESCRIPTION
**Pull request by Pradeep1308**
**Merge pre-release into Develop**

### CODE REVIEW

File: app/admin/RTMediaAdmin.php [hunk 1]
```diff
@@ -179,33 +179,33 @@ public function install_transcoder_admin_notice() {
...
+							'a' => array(
+								'href'   => array(),
+								'target' => array(),
+							),
+						);
+						echo wp_kses( __( 'Install <a href="https://wordpress.org/plugins/transcoder/" target="_blank">Transcoder plugin</a> to convert audio/video files and thumbnails generation.', 'buddypress-media' ), $allowed_tags );
+						?>
+					</p>
+				</div>
+				<script type="text/javascript">
```
Comment (robicse11127) [inline app/admin/RTMediaAdmin.php:196]: We can move these JS codes in to a separate/existing JS file to make the php code clean.

File: app/admin/RTMediaAdmin.php [hunk 2]
```diff
@@ -358,15 +357,15 @@ public function rtmedia_premium_addon_notice() {
 					</p>
 				</div>
 				<script type="text/javascript">
```
Comment (robicse11127) [inline app/admin/RTMediaAdmin.php:359]: We can move these JS codes in to a separate/existing JS file to make the php code clean.

File: app/admin/RTMediaAdmin.php [hunk 3]
```diff
@@ -1229,17 +1227,14 @@ public static function get_current_tab() {
...
 			<div class="wrap bp-media-admin <?php echo esc_attr( $this->get_current_tab() ); ?>">
 				<div id="icon-buddypress-media" class="icon32"><br></div>
 				<div>
 					<h2 class="nav-tab-wrapper"><?php $this->rtmedia_tabs(); ?>
 						<span class="<?php echo esc_attr( $align ); ?> by">
-							<a class="rt-link"
-								href="https://rtmedia.io/?utm_source=dashboard&utm_medium=plugin&utm_campaign=buddypress-media"
-								target="_blank"
-								title="rtCamp : <?php esc_attr_e( 'Empowering The Web With WordPress', 'buddypress-media' ); ?>">
+							<a class="rt-link" href="https://rtmedia.io/?utm_source=dashboard&utm_medium=plugin&utm_campaign=buddypress-media" target="_blank" title="rtCamp : <?php esc_attr_e( 'Empowering The Web With WordPress', 'buddypress-media' ); ?>">
```
Comment (robicse11127) [inline app/admin/RTMediaAdmin.php:1237]: Do not use raw url. Use `esc_url()` to escape the url.

File: app/admin/RTMediaAdmin.php [hunk 4]
```diff
@@ -1254,51 +1249,51 @@ public function render_page( $page_name, $option_group = null ) {
...
+										RTMediaFormHandler::rtForm_settings_tabs_content( $page_name, $sub_tabs );
+										echo '</div>';
+									} else {
+										do_settings_sections( $page_name );
+									}
+									?>
+
+									<div class="rtm-button-container bottom">
+										<div class="rtm-social-links alignleft">
+											<a href="http://twitter.com/rtMediaWP" class="twitter" target="_blank">
```
Comment (robicse11127) [inline app/admin/RTMediaAdmin.php:1279]: Do not use raw url. Use esc_url() to escape the url.

File: app/admin/RTMediaAdmin.php (hunk 1 above)
Comment (Pradeep1308) [reply app/admin/RTMediaAdmin.php:196]: as of now we are not changing code structure for this release

**Review by robicse11127 — APPROVED** [review]
As per discussion approving the PR with some escaping functions are needed to be applied

**Pull request merged**